ContactBook/Python/Agenda.py

```diff
@@ -0,0 +1,83 @@
+from time import sleep
+from os import system, name
+    
+from lib.file import *
+
+def header(txt):
+  lines = '-' * 30
+  print(lines)
+  print(txt.center(30))
+  print(f'{lines}\n')
+
+def menu(options):
+  header('Contact book')
+
+  count = 1
+  for option in options:
+    print(f'{count} -> {option}')
+    count += 1
```

Comment:
This would be a good place to use `enumerate()`  https://docs.python.org/3/library/functions.html